.github/workflows/release.yml

```diff
@@ -5,28 +5,28 @@ on:
       version:
         type: string
         description: version to tag image
-        
+
 jobs:
   release:
     runs-on: ubuntu-latest
     env:
       VERSION: ${{ github.event.inputs.version }}
     steps:
       - name: Setup - checkout
-        uses: actions/checkout@v3
-        
+        uses: actions/checkout@v4
+
       - name: Set up Go 1.x
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v4
         with:
-          go-version: ^1.13
+          go-version: ^1.21
         id: go
 
       - name: Login to DockerHub
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
       - name: docker
-        shell: bash    
+        shell: bash
         run: docker/docker-build.sh -v ${VERSION}
```

Comment:
docker-build.sh still uses the two shell scripts you replaced with the makefile. Also make sure that script still even works, I had to get rid of it in go-clouddriver because it kept erroring so I just used the `docker/build-push-action` instead

Reply:
<img width="1047" alt="image" src="https://github.com/homedepot/arcade/assets/43426700/c7a01313-a1ee-4454-a060-b8b55d6b657c">
The script is now working aside from pushing (I am not authenticated). Would you recommend dropping this script regardless in favor of the build-push-action?

Reply:
I only dropped the script in go-clouddriver because I couldn't get it to work but I think it's fine either way so it's up to you